### PR TITLE
Better user agent for clients

### LINKED ISSUE #65: Improve HTTP user_agent
deepocli should properly identify itself via the user_agent.

# Where
- call to api.deepomatic.com via deepomatic-client-python should use the already existing `Client` `user_agent_suffix` parameter
- call to studio.deepomatic.com should replace `deepomatic-client-python/(vesta)` by `deepocli/xxx` 

# What
- the string `deepocli`
- the deepocli version (maybe with git revision too if possible, would help for development versions)

### PR DESCRIPTION
**Pull request by maingoh**
Close #65

### CODE REVIEW

File: deepomatic/cli/cmds/feedback.py [hunk 1]
```diff
@@ -10,6 +10,7 @@
 from .studio_helpers.task import Task
 from ..common import TqdmToLogger, Queue
 from ..thread_base import Pool, MainLoop
+from ..version import __title__, __version__
```
Comment (thomas-riccardi) [inline deepomatic/cli/cmds/feedback.py]: not needed?

File: deepomatic/cli/cmds/studio_helpers/http_helper.py [hunk 2]
```diff
@@ -23,12 +24,14 @@ def __init__(self, token, verify, host, check_query_parameters, user_agent_suffi
...
         user_agent_params = {
+            'package_title': __title__,
+            'package_version': __version__,
             'requests_version': requests.__version__,
             'python_version': python_version,
             'platform': platform.platform()
         }
 
-        self.user_agent = 'deepomatic-client-python/(vesta) requests/{requests_version} python/{python_version} platform/{platform}\
+        self.user_agent = '{package_title}/{package_version} requests/{requests_version} python/{python_version} platform/{platform}\
```
Comment (thomas-riccardi) [inline deepomatic/cli/cmds/studio_helpers/http_helper.py]: is `__title__` standard and a good idea?

File: deepomatic/cli/cmds/feedback.py (hunk 1 above)
Comment (maingoh) [reply deepomatic/cli/cmds/feedback.py]: oupsy

File: deepomatic/cli/cmds/studio_helpers/http_helper.py (hunk 2 above)
Comment (maingoh) [reply deepomatic/cli/cmds/studio_helpers/http_helper.py]: there is no standard in packaging, though I inspire myself from https://github.com/kennethreitz/requests/blob/a79a63390bc963e5924021086744e53585566307/requests/__version__.py#L5

File: deepomatic/cli/cmds/studio_helpers/http_helper.py [hunk 3]
```diff
@@ -6,6 +6,7 @@
 import json
 from six import string_types
 from requests.structures import CaseInsensitiveDict
+from ...version import __title__, __version__
```
Comment (thomas-riccardi) [inline deepomatic/cli/cmds/studio_helpers/http_helper.py]: maybe import the module `deepomatic.cli` itself? it also has the version directly

**Pull request merged**